doc/tutorials/06-active_matter/SOLUTIONS/rectification_simulation.py

```diff
@@ -82,20 +83,20 @@ def a2quat(phi,theta):
 # Setup the box (we pad the diameter to ensure that the LB boundaries
 # and therefore the constraints, are away from the edge of the box)
 
-length      = 100
-diameter    = 20
-prod_steps  = 500
+length = 100
+diameter = 20
+prod_steps = 500
 prod_length = 500
-dt          = 0.01
+dt = 0.01
 
 # Setup the MD parameters
 
 system = espressomd.System(box_l=[length, diameter + 4, diameter + 4])
-system.set_random_state_PRNG()
+system.box_l = [length, diameter + 4, diameter + 4]
```

Comment:
You deleted the RNG seeding and set the box length twice. Also in some of the other files.